JuliaInterface/example/function_perform.gi

```diff
@@ -0,0 +1,48 @@
+LoadPackage( "JuliaInterface" );
+
+dirs:= DirectoriesPackageLibrary( "JuliaInterface", "example" );
+
+JuliaIncludeFile( Filename( dirs, "function_perform.jl" ) );
+
+if not IsBound(_JuliaFunctionByModuleNoConv) then
+  _JuliaFunctionByModuleNoConv:= _JuliaFunctionByModule;
+  _JuliaFunctionNoConv:= _JuliaFunction;
+fi;
+
+myfunc := JuliaFunction("myfunc", "Stuff");
+myfuncNoConv := _JuliaFunctionByModuleNoConv("myfunc", "Stuff");
+myfuncC := JuliaBindCFunction( "Stuff.myfunc", 2, ["a", "b"] );
+
+blafunc := JuliaFunction("blafunc");
+blafuncNoConv := _JuliaFunctionNoConv("blafunc");
+blafuncC := JuliaBindCFunction( "blafunc", 2, ["a", "b"] );
+
+GASMAN("collect"); ListX([1..10^5], [1..10], {i,j} -> i);; time;
+GASMAN("collect"); ListX([1..10^5], [1..10], ReturnFirst);; time;
+GASMAN("collect"); ListX([1..10^5], [1..10], myfunc);; time;
+GASMAN("collect"); ListX([1..10^5], [1..10], blafunc);; time;
+# GASMAN("collect"); ListX([1..10^5], [1..10], myfuncC);; time;
+GASMAN("collect"); ListX([1..10^5], [1..10], blafuncC);; time;
+GASMAN("collect"); ListX([1..10^5], [1..10], myfuncNoConv);; time;
+GASMAN("collect"); ListX([1..10^5], [1..10], blafuncNoConv);; time;
```

Comment:
Those `time;` statements are only valid in the interpreter; putting them into `.g` or `.gi` file is not really appropriate.

Reply:
I will leave them in for now. I would prefer to have two versions of this file
1. a testfile
2. a version that I can copy&paste for demo proposes